Geometry/TrackerGeometryBuilder/python/trackerParameters_cff.py

```diff
@@ -0,0 +1,9 @@
+import FWCore.ParameterSet.Config as cms
+
+from Geometry.TrackerGeometryBuilder.trackerParameters_cfi import *
+
+from Configuration.ProcessModifiers.dd4hep_cff import dd4hep
+
+dd4hep.toModify(trackerParameters,
+                fromDD4Hep = cms.bool(True),
```

Comment:
Does this line enable DD4hep Tracker parameters for all Extended 2021 workflows? We are not ready for that yet.

Reply:
No, only when a dd4hep Era is selected